app/assets/stylesheets/components/_personal-key.scss

```diff
@@ -13,6 +13,7 @@
   &::after {
     color: $silver;
     content: '-';
+    font-size: 1.5rem;
```

Comment:
Could we avoid this by setting the font-size on the `separator-text` element instead of the individual fragments of the code? Either something like `<div class="separator-text font-body-xl ...">` or `.separator-text { font-size: 1.5rem; }`.